build-tools/packages/build-cli/src/library/repoPolicyCheck/npmPackages.ts

```diff
@@ -1178,7 +1181,8 @@ export const handlers: Handler[] = [
 			}
 
 			const jestConfigFile = path.join(packageDir, jestFileName);
-			const config = (await import(path.resolve(jestConfigFile))) as { reporters?: unknown };
```

Comment:
I don't know why the import approach didn't work here. Maybe when the jest config itself calls require, it's not defined when the module is imported via import, but it is when the module is imported via require?